docs/Guide/application-commands/application-command-registry/acquiring-application-command-registry.mdx

```diff
@@ -3,51 +3,65 @@ sidebar_position: 2
 title: Acquiring an Application Command Registry
 ---
 
-To do anything with the `Application Command Registry` we need to acquire the one for the use. It can be done in two
-ways:
+To do anything with the [`Application Command Registry`][acr] we need to "acquire" one. To "acquire" an
+[`Application Command Registry`][acr] means to retrieve it from the Sapphire code. This may sound more complicated than
```

Comment:
```suggestion
[`Application Command Registry`][acr] means to retrieve an already existing one for the command name you specify or generating a brand new one. This may sound more complicated than
```